src/main/java/io/dockstore/provision/DOSPlugin.java

```diff
@@ -0,0 +1,67 @@
+package io.dockstore.provision;
+
+import com.google.common.collect.Lists;
+import org.apache.commons.lang3.StringUtils;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import ro.fortsoft.pf4j.*;
```

Comment:
Avoid star imports (this can usually be configured in your IDE)
http://seanmonstar.com/post/708954358/import-star-considered-harmful